plugins/node/opentelemetry-plugin-mongoose/jasmine.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
we currently use mocha for all of our tests so i think those tests would need to be converted (not sure if its a hard requirement though)